dataset/src/integrationTest/kotlin/com/cosmotech/dataset/service/DatasetServiceIntegrationTest.kt

```diff
@@ -809,64 +809,82 @@ class DatasetServiceIntegrationTest : CsmRedisTestBase() {
 
   @Test
   fun `link workspace from dataset`() {
+
+    assertNull(
+        datasetApiService
+            .findDatasetById(organizationSaved.id!!, datasetSaved.id!!)
+            .linkedWorkspaceIdList)
+
+    datasetApiService.linkWorkspace(organizationSaved.id!!, datasetSaved.id!!, workspaceSaved.id!!)
+
+    checkLinkedWorkspaceId()
```

Comment:
can it have a more specific name.
like checkOnlyOneLinkedWorkspaceId() or better use parameters same name but with a list of id to check if in the list.